### DISCUSSION TITLE
Import via URI

### DISCUSSION THREAD

**Discussion by collateral127** [2fast-team/2fast#79] [Ideas]
is there any chance we can get a feature added to import via the URI so we can pull that from the QR code and import that way?

**Comment by jp-weber** [comment]
You mean the function that we can pass a command line argument to the app in Uri style? Like ```twofastauth://uri/set?otpauth...``` That would be an excellent idea 😃

**Reply by collateral127** [reply]
thanks! I tried exporting some of my stuff via the qr code via Aegis, but my webcam was struggling a bit (it's old and rubbish) so i ended up exporting the URIs and parsing the information in them manually, so an automated way to do it would be brilliant :)
